### ISSUE TITLE
User test PRs are hard to parse and work through for individual PRs

### ISSUE THREAD

**Issue by DanielRosenwasser** [microsoft/TypeScript#37221]
Was chatting with @ahejlsberg today and we were commiserating about how difficult it is to understand user test PRs generated from @typescript-bot. (e.g. `user test this`).

To me, the issue is that the workflow just feels bad.

1. I'm told that the user tests _failed_ (always with the italics!)
1. I open up the user tests PR, which is slow because GitHub is slow
1. I open up the files view, which is slow because GitHub is slow
1. The differences are often just the deltas between today and yesterday's baselines.

Some ideas:

1. We only update user tests on a weekly basis. This avoids the noise and means someone doesn't have to update the differences every day.
1. Maybe diffs can be posted on the PR itself, within a `<details>` tag, e.g.

   `````markdown
   <details>
   <summary>
     The user tests <em>failed!</em>.
     Click here to view the diff of the user tests.
   </summary>

   ```diff
     derp
   - blop
   + bloop
   ```
   </details>
   `````

**Comment by ahejlsberg** [comment]
What I really want to see is the *delta* between a run on master and a run on the particular PR. As things stand now, I have no idea whether issues are preexisting conditions in master or caused by the PR, and I have no easy way to find out. I feel like I'm wasting quite a bit of time attempting to unravel this and it definitely slows down the bug fixing process.

**Comment by weswigham** [comment]
Hm, if the RWC workflow has proven to be OK, I can change the user tests to use a single known branch for `master` baseline updates (like RWC), then diff PR branches against that one (but it does mean you wouldn't be able to preemptively merge the changes into your pr; but i don't think people have really been doing that, anyway).
